src/jp2image.cpp

```diff
@@ -23,7 +23,6 @@
 */
 
 // *****************************************************************************
-// #define DEBUG
```

Comment:
Note that I get rid of these handmade definitions. We can pass the DEBUG flag through CMake easily